# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptTrace.h
# Commit: 7d5683180ac3
# Full Hash: 7d5683180ac3f9e7cc96d68dde27b8b48642a7ac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r a48aff5f2555 -r 7d5683180ac3 dom/script/ScriptTrace.h
--- a/dom/script/ScriptTrace.h	Thu Feb 17 15:57:52 2022 +0000
+++ b/dom/script/ScriptTrace.h	Thu Feb 17 15:57:52 2022 +0000
@@ -17,25 +17,26 @@
 // This macro is used to wrap a tracing mechanism which is scheduling events
 // which are then used by the JavaScript code of test cases to track the code
 // path to verify the optimizations are working as expected.
-#define TRACE_FOR_TEST(elem, str)                                              \
-  PR_BEGIN_MACRO                                                               \
-  nsresult rv = NS_OK;                                                         \
-  rv =                                                                         \
-      script::TestingDispatchEvent(elem, NS_LITERAL_STRING_FROM_CSTRING(str)); \
-  NS_ENSURE_SUCCESS(rv, rv);                                                   \
+#define TRACE_FOR_TEST(elem, str)                  \
+  PR_BEGIN_MACRO                                   \
+  nsresult rv = NS_OK;                             \
+  rv = mozilla::dom::script::TestingDispatchEvent( \
+      elem, NS_LITERAL_STRING_FROM_CSTRING(str));  \
+  NS_ENSURE_SUCCESS(rv, rv);                       \
   PR_END_MACRO
 
-#define TRACE_FOR_TEST_BOOL(elem, str)                                         \
-  PR_BEGIN_MACRO                                                               \
-  nsresult rv = NS_OK;                                                         \
-  rv =                                                                         \
-      script::TestingDispatchEvent(elem, NS_LITERAL_STRING_FROM_CSTRING(str)); \
-  NS_ENSURE_SUCCESS(rv, false);                                                \
+#define TRACE_FOR_TEST_BOOL(elem, str)             \
+  PR_BEGIN_MACRO                                   \
+  nsresult rv = NS_OK;                             \
+  rv = mozilla::dom::script::TestingDispatchEvent( \
+      elem, NS_LITERAL_STRING_FROM_CSTRING(str));  \
+  NS_ENSURE_SUCCESS(rv, false);                    \
   PR_END_MACRO
 
-#define TRACE_FOR_TEST_NONE(elem, str)                                     \
-  PR_BEGIN_MACRO                                                           \
-  script::TestingDispatchEvent(elem, NS_LITERAL_STRING_FROM_CSTRING(str)); \
+#define TRACE_FOR_TEST_NONE(elem, str)            \
+  PR_BEGIN_MACRO                                  \
+  mozilla::dom::script::TestingDispatchEvent(     \
+      elem, NS_LITERAL_STRING_FROM_CSTRING(str)); \
   PR_END_MACRO
 
 static nsresult TestingDispatchEvent(nsIScriptElement* aScriptElement,